utils_test.py

```diff
@@ -172,6 +172,7 @@ def test_vfs_construct_path(self):
         self.assertEqual(p, '/a/b/')
 
     def test_vfs_normpath(self):
+        """Test vfs_normpath method."""
```

Comment:
@vibhor98 @apb7 I thought we decided to exclude for test files from requiring docstrings?

Reply:
@seanlip: Yes, it would be better to exclude test files from requiring docstrings.

Reply:
Right. So why is this PR titled "Fix part of #4374: Add docstrings to utils_test.py" if we don't need docstrings for utils_test.py?

Reply:
@seanlip  yes we've decided to do so. That's why this file is not in the todo list of the issue. But here he has added docstrings to multiple files including the test file. May be that's why the PR is titled so.

Reply:
I checked Issue #4374. This file is not stated there. We have a file `test_utils.py` which needs to be documented and not `utils_test.py`. Also, it seems that @yashLadha has not been assigned any file from the list.

Reply:
Ah, OK, thanks for the clarification. @yashLadha, perhaps you could explain -- was this a miscommunication?

Reply:
I saw that the file is not documented and run the pylint check as mentioned in the gitter channel, and it gave the lines that needs docstring so i added them. @seanlip . That's why i have added this.  I was not aware that test files are excluded from pylint check. @vibhor98